client/src/route/auth/AccountTabData.tsx

```diff
@@ -1,13 +1,56 @@
-import { ITabs } from 'route/IData';
+import * as React from 'react';
+import { useAuth } from 'react-oidc-context';
+import { TabData } from 'components/tab/subcomponents/TabRender';
 
-const tabs: ITabs[] = [
+function ProfileTab() {
+  const { user } = useAuth();
+  const name = (user?.profile.preferred_username as string | undefined) ?? "";
+  const pfp = user?.profile.picture;
+  const profileUrl = user?.profile.profile;
+  const groups = ((user?.profile.groups as string[] | undefined) ?? []);
+
+  const boldName = React.createElement("b", null, name);
+  const boldGroups = groups.map(group => React.createElement("b", null, group));
+  const groupListing: React.ReactNode[] = [];
+  boldGroups.slice(0, -1).forEach((groupElement, i) => {
+    const isSecondToLastGroup = (i < boldGroups.length - 2);
+    const seperator = isSecondToLastGroup ? ", " : " and "
+    groupListing.push(groupElement, seperator);
+  });
+  groupListing.push(boldGroups.slice(-1));
```

Comment:
The [Array.splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) method shortens this code fragment. You can use

groupListing.splice(groupListing.length - 1, 0, 'and');